stac_fastapi/pgstac/stac_fastapi/pgstac/core.py

```diff
@@ -24,6 +27,7 @@ class CoreCrudClient(AsyncBaseCoreClient):
     async def all_collections(self, **kwargs) -> List[Collection]:
```

Comment:
I think, this needs to be `Collections` instead of `List[Collection]` as type returned.